RocketCast/DownloadBridge.swift

```diff
@@ -11,19 +11,17 @@ import Foundation
 import UIKit
 
 protocol DownloadBridgeProtocol {
-    func downloadPodcastXML(url:PodcastWebURL, result:(url: PodcastStorageURL) -> ())
+    func downloadPodcastXML(rssData:NSData, result:(podcast: PodcastModel) -> ())
```

Comment:
Should not be an NSData. Keep it as a URL for simplicity purposes. Also, I think this is Milton and Mo's territory